.github/workflows/update-insecure-dependencies.yaml

```diff
@@ -0,0 +1,75 @@
+name: "Update insecure dependencies"
+
+on:
+  workflow_dispatch: { }
+  schedule:
+    - cron: 0 8 * * *
+jobs:
+  update-insecure-dependencies:
+    strategy:
+      matrix:
+        branch:
+          - "release-2.0"
+          - "release-1.8"
+          - "release-1.7"
+          - "release-1.6"
+          - "release-1.5"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate GitHub app token
+        id: github-app-token
+        uses: tibdex/github-app-token@021a2405c7f990db57f5eae5397423dcc554159c # v1.7.0
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
```

Comment:
Mostly concerned about this, because on my fork I did not create an app to do a PR. I copied it from `update-changelog` - is this reusable?

Reply:
Yes it is reusable